Vagrantfile

```diff
@@ -46,6 +46,28 @@ Vagrant.configure('2') do |config|
     end
   end
 
+  config.vm.define :windows5 do |windows|
+    windows.vm.box = "jacqinthebox/windowsserver2016core"
+    windows.vm.guest = :windows
+    windows.vm.communicator = "winrm"
+    windows.vm.network :forwarded_port, guest: 5985, host: 35985, host_ip: '127.0.0.1', id: 'winrm'
+    windows.vm.provision 'shell', privileged: true, inline: 'slmgr /rearm'
+    windows.vm.provider "virtualbox" do |vb|
+      vb.gui = false
+    end
+  end
+
+  config.vm.define :windows6 do |windows|
+    windows.vm.box = "jacqinthebox/windowsserver2016core"
```

Comment:
I am connected to the ethernet at the office and downloading this image has been in progress for almost an hour. Estimated 5% done with projected 32 hours left. I guess I will just let it go over-night and test it out tomorrow.